dsp_permissions_scripts/utils/authentication.py

```diff
@@ -15,9 +15,6 @@ def _get_login_credentials(host: str) -> tuple[str, str]:
     if host == Hosts.LOCALHOST:
         user = "root@example.com"
         pw = "test"
-    elif host == Hosts.RDU_STAGE:
-        user = os.getenv("RDU_STAGE_EMAIL") or ""
-        pw = os.getenv("RDU_STAGE_PASSWORD") or ""
```

Comment:
rdu.dasch.swiss is mirrored from prod, so it isn't necessary to handle it separately. It will be handled by the `else`